From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: celenity <celenity@celenity.dev>
Date: Sun, 05 Jul 2026 20:19:20 +0000
Subject: [PATCH] Fenix - Increase update frequency

Increases the rate at which Firefox syncs with Remote Settings, from every 24 hours to hourly, and the rate at which
Firefox checks for add-on updates, from every 12 hours to hourly.

Signed-off-by: celenity <celenity@celenity.dev>
---
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Components.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Components.kt
index 87a4f911c0cf..80e9e7346880 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Components.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Components.kt
@@ -210,7 +210,7 @@ class Components(private val context: Context) {
 
     @Suppress("MagicNumber")
     val addonUpdater by lazyMonitored {
-        DefaultAddonUpdater(context, Frequency(12, TimeUnit.HOURS), notificationsDelegate)
+        DefaultAddonUpdater(context, Frequency(1, TimeUnit.HOURS), notificationsDelegate)
     }
 
     @Suppress("MagicNumber")
@@ -225,7 +225,7 @@ class Components(private val context: Context) {
     val remoteSettingsSyncScheduler by lazyMonitored {
         DefaultRemoteSettingsSyncScheduler(
             context,
-            Frequency(2, TimeUnit.HOURS),
+            Frequency(1, TimeUnit.HOURS),
         )
     }
 
-- 
